index.html

```diff
@@ -57,5 +59,204 @@ <h1>
+          <img src="./images/snapshot-2.svg" alt="" />
+        </div>
+        <div class="contain">
+          <p class="snapshot-title">
+            Multi-Post <br />
+            Stories
+          </p>
+          <div class="client-info">
+            <p class="client">CANOPY</p>
+            <div class="circle"></div>
+            <p>Back End Dev</p>
+            <div class="circle"></div>
+            <p>2015</p>
+          </div>
+          <p class="project-content">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="used-language">
+            <li class="language">html</li>
+            <li class="language">css</li>
+            <li class="language">javascript</li>
+          </ul>
+          <div href="#" class="see-project"><a>See Project</a></div>
+        </div>
+      </div>
+
+      <!-- Project 3 -->
+
+      <div class="work-section">
+        <div class="snapshot">
+          <img src="./images/snapshot-3.svg" alt="" />
+        </div>
+        <div>
+          <p class="snapshot-title">Tonic</p>
+          <div class="client-info">
+            <p class="client">CANOPY</p>
+            <div class="circle"></div>
+            <p>Back End Dev</p>
+            <div class="circle"></div>
+            <p>2015</p>
+          </div>
+          <p class="project-content">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="used-language">
+            <li class="language">html</li>
+            <li class="language">css</li>
+            <li class="language">javascript</li>
+          </ul>
+          <div href="#" class="see-project"><a>See Project</a></div>
+        </div>
+      </div>
+
+      <!-- Project 4 -->
+
+      <div class="work-section">
+        <div class="snapshot">
+          <img src="./images/snapshot-4.svg" alt="" />
+        </div>
+        <div>
+          <p class="snapshot-title">
+            Multi-Post <br />
+            Stories
+          </p>
+          <div class="client-info">
+            <p class="client">CANOPY</p>
+            <div class="circle"></div>
+            <p>Back End Dev</p>
+            <div class="circle"></div>
+            <p>2015</p>
+          </div>
+          <p class="project-content">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="used-language">
+            <li class="language">html</li>
+            <li class="language">css</li>
+            <li class="language">javascript</li>
+          </ul>
+          <div href="#" class="see-project"><a>See Project</a></div>
+        </div>
+      </div>
+    </div>
+    <!-- Work Section Ends -->
+
+    <!-- About Section Starts -->
+
+    <section class="about-section">
+      <h1>
+        About <br />
+        Myself
+      </h1>
```

Comment:
- [Optional[ Consider replacing this h1 tag with another heading tag (between h2 - h5). It is best SEO practice to have just a single h1 element in an entire HTML file.

Reply:
done